libc/newhdrgen/yaml/math.yaml

```diff
@@ -2291,38 +2398,59 @@ functions:
       - stdc
     return_type: int
     arguments:
-      - type: const double *
-      - type: const double *
+      - type: double *
+      - type: double *
```

Comment:
`const` is needed to match the signatures in the C23 standard. See section F.10.12.1 of https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3096.pdf.